perllib/FixMyStreet/App/Controller/Questionnaire.pm

```diff
@@ -215,6 +226,7 @@ sub submit_standard : Private {
                 anonymous    => $problem->anonymous,
             }
         );
+        $update->set_extra_metadata('questionnaire_as_comment', 'Reporter says ' . $c->stash->{status_comment} . ' via questionnaire' ) if $c->stash->{status_comment};
```

Comment:
This doesn't work for translation purposes, I'm afraid, you cannot concatenate substrings like this. I'd rather not store a string in the DB, makes it hard to change in future. Do you think this is needed? I thought the existing message worked okay, even if the problem state didn't update. "Still open, via questionnaire" makes sense as a comment, I think, and we don't need to worry about the fixed way round I think? Happy to chat about it.